### ISSUE TITLE
Running into error when tailing

### ISSUE THREAD

**Issue by mhahn** [remind101/stacker#151]
keep getting:

```
ssl.SSLError: [Errno 1] _ssl.c:1429: error:1408F119:SSL routines:SSL3_GET_RECORD:decryption failed or bad record mac
```

**Comment by phobologic** [comment]
https://github.com/kennethreitz/requests/issues/1906 is likely the cause here

**Comment by phobologic** [comment]
Not requests itself, but the underlying python 2.7 issue with certain SSL endpoints.

**Comment by phobologic** [comment]
BTW, this may be fixed in boto3/botocore.  For a long time I've thought about moving over, maybe this will be the impetus.

**Comment by phobologic** [comment]
I think this is probably fixed by #208  - if not, reopen.
